pkg/controller/deployer/deployer_controller.go

```diff
@@ -28,10 +28,12 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
+	"github.com/IBM/deployer-operator/pkg/apis/app/v1alpha1"
```

Comment:
is this a duplicated import of the next line ?

Reply:
yes, fixing it with next commit